acceptance/analyzer_test.go

```diff
@@ -36,9 +36,17 @@ var (
 	daemonOS, daemonArch string
 	readOnlyRegistry     *ih.DockerRegistry
 	authRegistry         *ih.DockerRegistry
+	customRegistry       *ih.DockerRegistry
 	registryNetwork      string
 )
 
+const (
+	readWriteImage = "image-readable-writable"
+	onlyReadImage  = "image-readable"
+	onlyWriteImage = "image-writable"
+	noAccessImage  = "noAccessImage"
```

Comment:
The same comment as https://github.com/buildpacks/imgutil/pull/125/files#r671486105.